.travis.yml

```diff
@@ -1,12 +1,50 @@
 language: python
-sudo: enabled
+sudo: required
+dist: trusty
+addons:
+  ssh_known_hosts: localhost
+  postgresql: "9.5"
+cache:
+  pip: true
+  directories:
+    - assembl/static/js/node_modules
+    - assembl/static/widget/card/bower_components
+    - assembl/static/widget/creativity/bower_components
+    - assembl/static/widget/session/bower_components
+    - assembl/static/widget/share/bower_components
+    - assembl/static/widget/video/bower_components
+    - assembl/static/widget/vote/bower_components
+    - assembl/static2/node_modules
 before_install:
- - sudo apt-get -qq update
+ # The following line is needed, otherwise the set_file_permissions fab command yells error "Operation not permitted"
+ - sudo usermod -a -G www-data $USER
  - sudo apt-get install -y fabric git openssh-server
- - fab -c configs/develop.rc install_single_server
- - fab -c configs/develop.rc bootstrap_from_checkout
-install: true
+ - ssh-keygen -P '' -f ~/.ssh/id_rsa && cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
+ - echo "_extends = ./testing.rc" > configs/travis.rc; echo "_gitbranch = $TRAVIS_COMMIT" >> configs/travis.rc;
+ # The following line produces warning "uses weak digest algorithm" during its `apt-get update`
```

Comment:
Likely coming from one of the dependencies being stale/old?
https://askubuntu.com/questions/760796/how-to-fix-apt-signature-by-key-uses-weak-digest-algorithm-sha1